### DISCUSSION TITLE
ServiceRequest

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#14] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [ServiceRequest](http://hl7.org/fhir/R4/servicerequest.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core ServiceRequest, or if there is a need for an additional ServiceRequest profile in AU Core.

AU Core ServiceRequest profile would set minimum expectations for a ServiceRequest resource to record, search, and fetch information about requests, plans, or orders for a service for a patient. 

AU Core  ServiceRequest should be based on any relevant AU Base profiles and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the ServiceRequest resource when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context. .

**Comment by timothy-panoho** [comment]
Question - could this be used to call for IPS requests or is designed to specific documentation requests? In any case, I think this is important for AU core?

**Comment by heathfrankel** [comment]
In order for a requester to assign a referral to a fulfiller and allow them to accept, reject and report progress, a **Task** resource should be used. 
Please include the Task resource in AU Core R1 if the Service Request is included otherwise there will be implementations that are not following the recommended FHIR request workflow patterns.

**Reply by dt-r** [reply]
Agreed that a decision on if ServiceRequest is included in R1 needs to discuss relationship to Task and what cases are satisfied by defining minimum read and search around ServiceRequest.

**Comment by rachelhayhurst** [comment]
What is the relationship between service request and a provider directory - at the moment there is not a defined (coded) list of provider type for clinical information systems - these fields are free text and determined by the author of the address book record. Ideally this would relate to a nationally maintained provider directory
